api/src/functions/express/app.ts

```diff
@@ -211,7 +211,7 @@ const dynamicsHousingRegistrationStatusClient = DynamicsHousingRegistrationStatu
 });
 
 const taxClearanceCertificateClient = ApiTaxClearanceCertificateClient(logger, {
-  orgUrl: process.env.TAX_CLEARANCE_CERTIFICATE_URL || "",
+  orgUrl: process.env.TAX_CLEARANCE_CERTIFICATE_URL || "http://localhost:9000",
```

Comment:
[dust] The pattern we use in the rest of the file is:

```typescript
`http://${IS_DOCKER ? "wiremock" : "localhost"}:9000`
```

Would that apply here as well?

Reply:
I think it would be okay if it wasn't added as part of this commit and added when the cypress test is introduced.